udata_front/tests/views/test_oembed.py

```diff
@@ -26,139 +30,164 @@ class OEmbedAPITest:
-        assert 'width' in response.json
-        assert 'maxwidth' in response.json
-        assert 'height' in response.json
-        assert 'maxheight' in response.json
-        assert response.json['type'] == 'rich'
-        assert response.json['version'] == '1.0'
-        card = theme.render('dataset/card-xs.html', dataset=dataset)
-        assert card in response.json['html']
+        assert "html" in response.json
+        assert "width" in response.json
+        assert "maxwidth" in response.json
+        assert "height" in response.json
+        assert "maxheight" in response.json
+        assert response.json["type"] == "rich"
+        assert response.json["version"] == "1.0"
+        card = theme.render("dataset/card-xs.html", dataset=dataset)
+        assert card in response.json["html"]
 
     def test_oembed_for_dataset_with_organization(self, api):
-        '''It should fetch a dataset in the oembed format with org.'''
+        """It should fetch a dataset in the oembed format with org."""
         organization = OrganizationFactory()
         dataset = DatasetFactory(organization=organization)
 
-        url = url_for('api.oembed', url=dataset.external_url)
+        url = url_for("api.oembed", url=dataset.external_url)
 
-        response = api.get(url)
+        response = api.get(
+            url,
+            headers={"Origin": "http://localhost"},
+        )
         assert200(response)
         assert_cors(response)
 
-        card = theme.render('dataset/card-xs.html', dataset=dataset)
-        assert card in response.json['html']
+        card = theme.render("dataset/card-xs.html", dataset=dataset)
+        assert card in response.json["html"]
 
     def test_oembed_for_dataset_redirect_link(self, api):
-        '''It should fetch an oembed dataset using the redirect link.'''
+        """It should fetch an oembed dataset using the redirect link."""
         dataset = DatasetFactory()
-        redirect_url = url_for('datasets.show_redirect',
-                               dataset=dataset, _external=True)
-
-        url = url_for('api.oembed', url=redirect_url)
-        response = api.get(url)
+        redirect_url = url_for(
+            "datasets.show_redirect", dataset=dataset, _external=True
+        )
+
+        url = url_for("api.oembed", url=redirect_url)
+        response = api.get(
+            url,
+            headers={"Origin": "http://localhost"},
+        )
         assert200(response)
         assert_cors(response)
-        assert 'html' in response.json
-        assert 'width' in response.json
-        assert 'maxwidth' in response.json
-        assert 'height' in response.json
-        assert 'maxheight' in response.json
-        assert response.json['type'] == 'rich'
-        assert response.json['version'] == '1.0'
-        card = theme.render('dataset/card-xs.html', dataset=dataset)
-        assert card in response.json['html']
+        assert "html" in response.json
```

Comment:
I would not modify quotes in this PR since single quotes are still the norm.
We should probably wait for a global ruff pass?

Reply:
Done :-)